src/include/mallocMC/creationPolicies/Scatter_impl.hpp

```diff
@@ -921,8 +934,12 @@ namespace ScatterKernelDetail{
       __device__ unsigned getAvailableSlotsAccelerator(size_t slotSize){
         int linearId;
         int wId = threadIdx.x >> 5; //do not use warpid-function, since this value is not guaranteed to be stable across warp lifetime
-        uint32 activeThreads  = __popc(__ballot(true));
 
+#if(__CUDACC_VER_MAJOR__ >= 9)
+        uint32 activeThreads  = __popc(0xFFFFFFFF, __ballot(true));
```

Comment:
`__popc_sync`?

Reply:
no `__popc` is no warp function, there is no sync function available